applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig $(dvfs_state)_ENABLE
+	bool "DVFS module $(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if $(dvfs_state)_ENABLE
+
+config $(dvfs_state)_TIMEOUT_MS
+	int "timeout"
+	default 5000
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS module will go back to
+	  low frequency after entering $(dvfs_state).
```

Comment:
`.. after which DVFS request related to $(dvfs_state) times out. A request is automatically cleared on timeout.`.

You can also mention that value of `0` can be used here to disable the timeout (this must also be handled in implementation). Currently all of the request time out (which is not intended).

Reply:
we don't have 0 as disabling timeout for now so I don't add it.